docs/training-apis/api-core/reference.txt

```diff
@@ -0,0 +1,53 @@
+.. _core-reference:
+
+###############
+ API Reference
+###############
+
+..
+   XXX how do I document context.core.init()?
```

Comment:
Do you mean you are not sure how to tell sphinx to include the docstring for core._context.init()? I am no sphinx expert but it looks like you might be able to use automodule:: with explicitly specifying
```
:members: init
```